src/main/kotlin/github/GitHubImpl.kt

```diff
@@ -99,7 +99,7 @@ object GitHubImpl {
 
     fun versionExists(identifier: String, version: String): Boolean = microsoftWinGetPkgs
         .getDirectoryContent(GitHubUtils.getPackagePath(identifier))
-        ?.map(GHContent::name)
+        ?.map(GHContent::getName)
```

Comment:
The Kotlin compiler 1.9+ supports using `something` instead of `getSomething` in lambda references so `GHContent::name` will work fine

Reply:
This was also suggested by IntelliJ IDEA.